src/content/guides/farmlands-portal.mdx

```diff
@@ -3,11 +3,11 @@ title: Farmlands Portal Guide
 description: Farmlands Co-operative (Farmands) have partnered with Centrapay Payment Gateway (Centrapay) to deliver a secure web portal for Farmlands Card payments.
 nav:
   path: Connections/Farmlands
-  title: Portal Guide
+  title: Portal Introduction Guide
```

Comment:
Farmlands asked me to remove 'Introduction' from the title last week. Have they asked for it to be added back in?

Reply:
Got it, thanks